unit/api/transaction_resource.py

```diff
@@ -9,8 +9,8 @@ def __init__(self, api_url, token, retries):
         super().__init__(api_url, token, retries)
         self.resource = "transactions"
 
-    def get(self, transaction_id: str, include: Optional[str] = "") -> Union[UnitResponse[TransactionDTO], UnitError]:
-        response = super().get(f"{self.resource}/{transaction_id}", {"include": include})
+    def get(self, transaction_id: str, account_id: str, include: Optional[str] = "") -> Union[UnitResponse[TransactionDTO], UnitError]:
```

Comment:
we want the _get_ function to be backwards compatible so let's remove account_id changes.
Let's create another function for this case and call it **get_by_id_and_account** that will contain the new path you added f"accounts/{account_id}/{self.resource}/{transaction_id}"